### DISCUSSION TITLE
DIA_DIA-Umpire_SpecLib_Quant result output files

### DISCUSSION THREAD

**Discussion by cattlemer** [Nesvilab/FragPipe-Analyst-doc#43] [Q&A] [answered]
Hi all, I'm pretty new to FragPipe. I just did a big batch with DIA_DIA-Umpire_SpecLib_Quant workflow using both DDA and DIA data. What are the differences between results in protein.tsv versus report.pg_matrix versus the output to skyline (export report again). Are they all the same data? Or does the protein.tsv were generated from the DDA data? I'm asking this because sometimes, even if Dia-NN fail to run (e.g., ciRT calibration failed and no spectra library was built), there is still the protein.tsv peptid.tsv results generated. 
I guess ultimately, I wonder if these results (i.e., the protein.tsv) have any value or trustworthy even the whole workflow did not run completely.

**Comment by cattlemer** [comment]
FYI-my name is Wenting Li, post-doc at UC Davis. I totally forgot to introduce myself.

Thanks and excited to hear anyone's thoughts on this.

Wenting

**Comment by fcyu** [comment, marked as answer]
protein.tsv is from the identification, report.pg_matrix is from quantification. 

Best,

Fengchao

**Reply by cattlemer** [reply]
Thanks Fengchao,

What happens when a protien is ID-ed but not quantified? Is it because of low ion abundance? 

Best,
Wenting

**Reply by fcyu** [reply]
Mostly due to low abundance.
